src/app/modules/reports/components/time-entries-table/time-entries-table.component.spec.ts

```diff
@@ -123,18 +149,18 @@ describe('Reports Page', () => {
       const column = 3;
       expect(component.bodyExportOptions(durationTime, row, column, node)).toMatch(decimalValidator);
     });
-
+    
     it('The data should not be displayed as a multiple of hour when column is different of 3', () => {
       const column = 4;
       expect(component.bodyExportOptions(durationTime, row, column, node)).toBe(durationTime.toString());
     });
-
+    
     it('The link Ticket must not contain the ticket URL enclosed with < > when export a file csv, excel or PDF', () => {
       const entry = '<a _ngcontent-vlm-c151="" class="is-url">https://TT-392-uri</a>';
       const column = 0;
       expect(component.bodyExportOptions(entry, row, column, node)).toBe('https://TT-392-uri');
     });
-
+    
```

Comment:
extra blank spaces here, are these the line endings from windows?

Reply:
Thanks for your comment @almeida-erick . The blank spaces have been  deleted